### ISSUE TITLE
Use relative links for images in CSS

### ISSUE THREAD

**Issue by WebDevJL** [Puzzlout/TrekToursFlyer#93]
Damir,

Can we use relative links for images that we declare in CSS?

We have already many examples but when I have this kind of URL, the image url won't work: 
https://over-e-webdevjl.c9users.io/TrekToursFlyer/web/app_dev.php/en/product/1#section-review

The above URL is a dev box hosted by C9 for free and I can add vshost. My base URL is https://over-e-webdevjl.c9users.io/TrekToursFlyer and the images are not loaded because the URL return HTTP/404.

PS: the box is not running at the moment.

​Thanks for your input.​

**Comment by damirius** [comment]
@WebDevJL 

This one was kinda tricky. I fixed paths in css app-theme file to be relative to it, but problem persisted. Make sure that when you deploy it to production environment you run `php bin/console assetic:dump --env=prod --no-debug` so that it creates combined css file for production. For some reason it kept adding two "../" instead of one. I tested it with my local production site which is located on localhost/trektoursflyer/web/ instead in some root domain so it should work. But tell me if I missed something
